applications/nrf_desktop/README.rst

```diff
@@ -537,7 +537,8 @@ The peripheral supports one wireless connection at a time, but it can be bonded
    See the :ref:`nrf_desktop_bluetooth_guide_fast_pair` documentation section for details.
 
 The nRF Desktop Bluetooth Central device scans for all bonded peripherals that are not connected.
-The scanning is interrupted when any device connected to the dongle through Bluetooth comes in use.
+Right after entering scanning state, the scanning operation is uninterruptable for predefined time (:kconfig:option:`CONFIG_DESKTOP_BLE_FORCED_SCAN_DURATION_S`) to speed up connection establishment with Bluetooth Peripherals.
+After the timeout, the scanning is interrupted when any device connected to the dongle through Bluetooth comes in use.
```

Comment:
"comes in use" Can we be more specific here on what it means? HID reports exchange, etc.?

Reply:
ok, added clarification